docs/release-notes/8.0/8.0.35-31.0.md

```diff
@@ -0,0 +1,39 @@
+# Percona XtraBackup 8.0.35-31 (2024-06-18)
+
+Get started with [Quickstart Guide for Percona XtraBackup](..//..//quickstart-overview.md).
+
+## Supported versions
+
+Starting with Percona XtraBackup 8.0.34, you can take backups of Percona Server for MySQL 8.0.34, MySQL 8.0.34, and **higher versions within the 8.0 series**.
+
+Check the latest list of all [Supported versions](..//..//supported-versions.md).
+
+## Release highlights
+
+This release adds Percona XtraBackup ARM64 packages and provides bug fixes.
+
+## Bug fix
+
+* [PXB-3283]: If backup contains tables with [temporal datatypes] (https://dev.mysql.com/doc/refman/8.0/en/date-and-time-types.html) and their charset/collation changed (ALTER TABLE CONVERT CHARACTER SET) before the backup is taken, xtrabackup export couldn't generate the correct configuration (.cfg) files required by the import.
+
+* [PXB-2797]:  When you import a table with a full-text index from an XtraBackup backup, you might encounter a schema mismatch. This happens when the table's structure in the backup does not match the structure expected by the database.
+
+* [PXB-3210]: Percona XtraBackup version 8.0.33-28 and later fail to build on MacOS because of the FIND_PROCPS() function. This function causes issues during the build process, preventing the software from compiling successfully.
+
+* [PXB-3130]: when preparing 8.0.30 backups with newer xtrabackup versions, The prepare step failed. (Thank you to LiNing for the contribution!)
```

Comment:
when preparing -> When preparing
The prepare step failed - the prepare step failed
(Thank you to LiNing for the contribution!) -> (Thanks to LiNing for the contribution!)